ci-operator/step-registry/openshift-qe/kube-burner/openshift-qe-kube-burner-commands.sh

```diff
@@ -0,0 +1,65 @@
+#!/bin/bash
+set -o errexit
+set -o nounset
+set -o pipefail
+set -x
+cat /etc/os-release
+oc config view
+oc projects
+python --version
+pushd /tmp
+python -m virtualenv ./venv_qe
+source ./venv_qe/bin/activate
+
+ES_PASSWORD=$(cat "/secret/password")
+ES_USERNAME=$(cat "/secret/username")
+
+GSHEET_KEY_LOCATION="/ga-gsheet/gcp-sa-account"
+export GSHEET_KEY_LOCATION
+
+git clone https://github.com/cloud-bulldozer/e2e-benchmarking --depth=1
+pushd e2e-benchmarking/workloads/kube-burner-ocp-wrapper
+
+if [[ $WORKLOAD == *"cluster-density-v2"* ]]; then
+    current_worker_count=$(oc get nodes --no-headers -l node-role.kubernetes.io/worker=,node-role.kubernetes.io/infra!=,node-role.kubernetes.io/workload!= --output jsonpath="{.items[?(@.status.conditions[-1].type=='Ready')].status.conditions[-1].type}" | wc -w | xargs)
+    # Run a non-indexed warmup for scheduling inconsistencies
+    ES_SERVER="" ITERATIONS=${current_worker_count} CHURN=false ./run.sh
```

Comment:
lets also add in `--gc` to be true for the non indexed warmup, in case its false in the job. We'll always want to clean up the results on this one to be sure they dont interrupt with results in measurable run